programming/sublime-text/pspec.xml

```diff
@@ -34,7 +34,7 @@
             <Path fileType="data">/usr/share/icons/hicolor/*/sublime-text.png</Path>
         </Files>
         <Replaces>
-            <Package>sublime-text-3</Package>
+            <Package>sublime-text-4</Package>
```

Comment:
This shouldn't be changed. The "Replaces" section is used to replace an old, differently named version of this package. The old package was called `sublime-text-3`, before it was made version-independent. There is no `sublime-text-4` as far as I know (and if there is, it should be added to the list, not replace the other entry)